src/Umbraco.Core/Runtime/IRuntimeMinifier.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Umbraco.Core.Assets;
+
+namespace Umbraco.Core.Runtime
+{
+    public interface IRuntimeMinifier
+    {
+        string GetHashValue { get; }
```

Comment:
Method should be more descriptive as to what is doing... this is a cache buster value so should just be called CacheBuster